src/worldfactory.cpp

```diff
@@ -1148,8 +1159,8 @@ int worldfactory::show_worldgen_tab_modselection(WINDOW *win, WORLDPTR world)
             redraw_description = true;
             redraw_list = true;
             redraw_active = true;
-            draw_worldgen_tabs( win, 0 );
-            draw_modselection_borders( win, &ctxt );
+            //draw_worldgen_tabs( nwin, 0 );
```

Comment:
Ugh, please don't commit commented out code. Just remove it. If it is needed again later, one can look it up in the git history, or simply revert the removal commit.